doc/nrf/gs_programming.rst

```diff
@@ -98,8 +99,12 @@ it might return an error indicating a project load failure. For example::
 	The project file <filepath> is invalid.
 	The reported error is 'solution load command failed (1)'
 
-This issue might be caused by a variety of problems, such as incorrectly specified
-project file paths.
+This issue might be caused by a variety of problems, such as incorrectly specified project file paths.
 SES helps you to identify the source of the issue by providing a text output
 with detailed information about the error. Make sure to click **OK** on the
 error pop-up message and then inspect the text output in SES.
+
+.. tip::
+   SES uses the PATH variable to find executables.
+   If you get an error that a tool or command cannot be found, add its location to the PATH variable.
```

Comment:
@hakonfam @sigvartmh See https://devzone.nordicsemi.com/f/nordic-q-a/46173/can-t-flash-merge-hex-error - do we need to add a note that mergehex must be in the path?

Reply:
I think that could be a good idea after reading the support case.

Reply:
ok, not changed